cli/src/main/java/org/apache/iotdb/tool/AbstractCsvTool.java

```diff
@@ -99,6 +99,9 @@
         "yyyy/MM/dd'T'HH:mm:ss",
         "yyyy.MM.dd'T'HH:mm:ss"
       };
+  protected static final int CODE_OK = 0;
+  protected static final int CODE_ERROR = 1;
+
```

Comment:
These lines from come PR #5026. Consider updating export-csv.bat and import-csv.bat ?